starknet_py/hash/contract_class.py

```diff
@@ -2,10 +2,9 @@
 from typing import List
```

Comment:
Why this file is named `contract_class`?

Reply:
It cannot be called `class.py` as it's some internal name 😿

Reply:
I would name it `class_hash`. It is a very well-known name in StarkNet and import 
```from starknet.py.hash.class_hash import compute_class_hash```
isn't too bad

Reply:
@cptartur, you mentioned the repeating names in module path above, what do you think?

Reply:
I think it's okay in this case. It'd be worse if it was `hash.hash`.

Reply:
Let's name it as @war-in suggested.

Reply:
Okay, I will do it in next pr though